plugins/reporters/static-html/src/main/kotlin/TablesReportModelMapper.kt

```diff
@@ -128,6 +128,7 @@ private fun getProjectTable(input: ReporterInput, project: Project): ProjectTabl
     val projectIssuesForId = input.ortResult.dependencyNavigator.projectIssues(project)
     val scannerIssuesForId = input.ortResult.getScannerIssues()
     val scopesForId = input.ortResult.getScopesForDependencies(project)
+    val advisorIssuesForId = input.ortResult.getAdvisorIssues()
```

Comment:
Nit: I guess it would have been slightly more natural to move this line directly below `scannerIssuesForId`.